src/test/java/fr/univartois/sonargo/coverage/CoverageSensorTest.java

```diff
@@ -108,8 +109,31 @@ public void testExecution() {
 	    testerContext.fileSystem().add(new DefaultInputFile("myProjectKey", "pixel/pixel.go")
 		    .setLanguage(GoLanguage.KEY).initMetadata(sb.toString()));
 
+	    reader = new BufferedReader(new FileReader(
+		    new File(CoverageSensor.class.getResource("/coverage/mathutil/filewithnotcoverage.go").getFile())));
+
+	    sb = new StringBuilder();
+	    while ((sCurrentLine = reader.readLine()) != null) {
+		sb.append(sCurrentLine + "\n");
+	    }
+
+	    testerContext.fileSystem().add(new DefaultInputFile("myProjectKey", "mathutil/filewithnotcoverage.go")
+		    .setLanguage(GoLanguage.KEY).initMetadata(sb.toString()));
+
 	    sensor.execute(testerContext);
```

Comment:
filewithnotcoverage.go -> filewithnocoverage.go

Reply:
It's fix :)